Foundation/NSCFDictionary.swift

```diff
@@ -139,8 +143,12 @@ internal func _CFSwiftDictionaryContainsValue(dictionary: AnyObject, value: AnyO
 internal func _CFSwiftDictionaryGetKeysAndValues(dictionary: AnyObject, keybuf: UnsafeMutablePointer<Unmanaged<AnyObject>?>, valuebuf: UnsafeMutablePointer<Unmanaged<AnyObject>?>) {
     var idx = 0
```

Comment:
right, I thought this was sufficiently unlikely as to not bother with the special case. but I will fix it.